script.js

```diff
@@ -14,3 +70,490 @@ backMenu.forEach((element) => element.addEventListener('click', () => {
+          <li><a href="https://medium.com/BlessmanNewton"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+      </ul>
+  </div>
+  <hr class="hr_footer">
+  `;
+
+  body.appendChild(footer);
+  // ----contact section above -----
+
+  const view = document.querySelectorAll('.view-project');
+  view.forEach((element) => element.addEventListener('click', () => {
+    const show = document.createElement('div');
+    const detailPop = document.createElement('div');
+    show.className = 'show';
+    detailPop.className = 'pop';
+    detailPop.innerHTML = `
+                      <div class="popup-header">
+                        <h2 class="desktop">Keeping track of hundreds
+                         of components website</h2>
+                         <h2 class="mob-title">Multi Post Stories</h2>
+                        <span id="close">&times;</span>
+                      </div>
+                        <div class="popup-button">
+                          <button id="lists">HTML</button>
+                          <button id="lists">Boostrap</button>
+                          <button id="lists">Ruby on rails</button>
+                        </div>
+                      <div class="popup-body">
+                        <img class="desktop" src="${portfolioData[0].image[5]}" alt="">
+                        <img class="mob" src="${portfolioData[0].image[6]}" alt="">
+                        <div class="pop-details">
+                          <p>
+                          Lorem Ipsum is simply dummy text of the
+                          printing and typesetting industry.
+                          Lorem Ipsum has been the industry's
+                          standard dummy text ever since the 1500s,
+                          when an unknown printer took a
+                           galley of type and scrambled it 1960s.
+                          </p>
+
+                          <p class="desktop">
+                          Lorem Ipsum is simply dummy text of the
+                          printing and typesetting industry.
+                          Lorem Ipsum has been the industry's
+                          standard dummy text ever since the 1500s,
+                          when an unknown printer took a
+                           galley of type and scrambled it 1960s.
+                          </p>
+                         
+                          <div class="popup-button">
+                            <button class="popup-btn"><a href="${workData[0].live_see}">See live
+                            <i class="fa fa-external-link" aria-hidden="true"></i></a></button>
+                            <button class="popup-btn git"><a href="${workData[0].link}">See Source
+                                <i class="fa fa-github" aria-hidden="true">
+                                </i></a>
+                            </button>
+                          </div>
+                        </div>
+                      </div>
```

Comment:
Your design
![image](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/78251927/3da533fd-5330-4d94-8392-12ad1fb6b64e)

Figma
![image](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/78251927/374cd0a2-0ff2-4549-a555-733cdd9caa24)

Please Kindly modify the background of the popup so that it matches the template

   - According to the template, the background has blur effect 🛠
   - To ensure that we stick to the design as much as possible, kindly use the same `font` and `color`as [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).


Your design
![image](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/78251927/b4739c4b-2f44-46a8-8d5b-5a2c1362754d)

![image](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/78251927/b665dc00-c289-4bbb-8ac0-fd303cc55a4f)



Figma
![image](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/78251927/e42440a1-7549-4c02-9612-a75ac47a3ed7)

![image](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/78251927/c4618016-45b4-4d1a-a119-2d5690c80aa8)

Your design (Mobile)
![image](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/78251927/0e3d7f8c-8be7-4187-9c69-a76be0bacae5)

Figma
![image](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/78251927/7eefa699-3307-40c3-9b55-2af127b463af)

- Please let's adjust the spacing `margin/padding` on top, right, bottom, and left in the [Mobile & Desktop] design to ensure that the popup window is positioned at the center of the page to match the template as much as possible.